src/js/providers/tracks-mixin.js

```diff
@@ -51,9 +51,12 @@ function setTextTracks(tracks) {
         // Remove the 608 captions track that was mutated by the browser
         this._textTracks = _.reject(this._textTracks, function(track) {
             const trackId = track._id;
+            this._unknownCount = 0;
             if (this.renderNatively && trackId && trackId.indexOf('nativecaptions') === 0) {
                 delete this._tracksById[trackId];
                 return true;
+            } else if (track.name.indexOf('Unknown') === 0) {
```

Comment:
Why the `else if`?

Reply:
@egreaves I think there's another problem with this unknown count thing if we assign the count when we label the tracks and then set names on the tracks in track-helper:

```js
    if (!label) {
        label = 'Unknown CC';
        unknownCount += 1;
        if (unknownCount > 1) {
            label += ' [' + unknownCount + ']';
        }
    }
```
You basically shouldn't be counting these at all, you should be parsing `unknownCount` from the name. (if not found, it's 1, otherwise it's this pattern`/^Unknown CC(?: \[(\d+)\])?$/`.

Something like:
```js
if (track.name.indexOf('Unknown') === 0) {
    this._unknownCount = `/^Unknown CC(?: \[(\d+)\])?$/.match(0) || 1;`
}
```

I would like the description to explain things from the context of the bug fix a little "because the captions meny will say X instead of Y when there are Z 608 tracks".